idris-commands.el

```diff
@@ -289,10 +290,10 @@ Idris process. This sets the load position to point, if there is one."
           (setq idris-currently-loaded-buffer nil)
           (idris-switch-working-directory srcdir)
           (let ((result
-                 (if idris-load-to-here
-                     (idris-eval `(:load-file ,fn
-                                              ,(idris-get-line-num idris-load-to-here)))
-                   (idris-eval `(:load-file ,fn)))))
+                 (idris-eval
+                  (if idris-load-to-here
+                      `(:load-file ,fn ,(idris-get-line-num idris-load-to-here))
+                    `(:load-file ,fn)))))
```

Comment:
This now have the same structure as https://github.com/idris-hackers/idris-mode/blob/c244db74da243a623543b94b5429a10b864464d8/idris-commands.el#L230-L233 
making `idris-load-file` and `idris-load-file-sync` more similar and potentionally improved.